package.json

```diff
@@ -0,0 +1,78 @@
+{
+  "name": "libp2p-discv5",
+  "version": "0.0.1",
+  "description": "An implementation of Discovery V5 using Libp2p components",
+  "main": "index.js",
+  "files": [
+    "src",
+    "dist"
+  ],
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "lint": "tslint --project tsconfig.json",
+    "release": "aegir release",
+    "build": "aegir build",
+    "test": "mocha -r ts-node/register test/**/*.test.ts",
+    "test:node": "aegir test --target node",
+    "test:browser": "aegir test --target browser --target webworker",
+    "tsc": "tsc"
+  },
+  "pre-push": [
+    "lint"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ChainSafe/libp2p-discv5.git"
+  },
+  "keywords": [
+    "discovery",
+    "discv5",
+    "ethereum",
+    "libp2p",
+    "peer",
+    "p2p"
+  ],
+  "author": "ChainSafe Systems",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/ChainSafe/libp2p-discv5/issues"
+  },
+  "homepage": "https://github.com/ChainSafe/libp2p-discv5#readme",
+  "devDependencies": {
+    "@types/chai": "^4.2.0",
+    "@types/mocha": "^5.2.7",
+    "@types/node": "^12.0.10",
```

Comment:
@polkadot/ts might help with some of types down there

Reply:
Is @polkadot/ts up to date?

Reply:
Not sure but libp2p usually tries to be backwards compatible so api should be pretty much the same. I might be wrong, but ci might complain since strict is enabled in tsconfig